src/tasks/netcore/NetCoreTaskHelper.ts

```diff
@@ -208,25 +208,25 @@ export class NetCoreTaskHelper implements TaskHelper {
             const appVolume: DockerContainerVolume = {
                 localPath: path.dirname(helperOptions.appProject),
                 containerPath: runOptions.os === 'Windows' ? 'C:\\app' : '/app',
-                permissions: 'rw'
+                permissions: 'rw,z'

```

Comment:
I found one last issue; if Windows containers are used, the `,z` option is not valid. Can you add checks so that `,z` is only added for not-Windows containers? e.g.:
```suggestion
                permissions: runOptions.os === 'Windows' ? 'rw' : 'rw,z'
```

Reply:
That's a great idea, I like it!

Reply:
I've added the filter.

You may have some suggestions about the style/implementation. (I don't write TypeScript often.)

Reply:
Sorry for the slow response, it's been a very busy week. I'm taking today and next week off so I won't be able to get to this until after the US Thanksgiving holiday.

Reply:
@bwateratmsft it would be nice if this, and podman support (https://github.com/microsoft/vscode-docker/pull/3306) are part of the next release (`v1.19`).

I'd like to write a blog post about how use this plugin to debug a .NET app running in a container on Fedora.
Fedora uses podman and SELinux.

Reply:
> this, and podman support (#3306)

and https://github.com/microsoft/vscode-docker/issues/3281 also.

Reply:
Sorry again for the slow response. Last week was really hectic. I will get to this PR this week; we are planning to release 1.19 tentatively 17 Jan so this will definitely be a part of it.

Reply:
No problem. As the changes go in, I'll verify they work well on Fedora.